lib/src/cancelable_operation.dart

```diff
@@ -174,8 +178,11 @@ class CancelableCompleter<T> {
 
   /// Completes [operation] to [value].
   ///
-  /// If [value] is a [Future], this will complete to the result of that
-  /// [Future] once it completes.
+  /// If [value] is a Future the [operation] will complete with the result of
```

Comment:
`Future` -> `[Future]`

(Always link source names the first time they occur in a doc, at least code-quote them if they occur more than once).